CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing Guidelines
+
+Thank you for considering contributing to our open source application!
+We welcome any and all contributions, and appreciate the time and effort
+you put into making our project better.
+
+## How to Contribute
+
+1. Fork the project and create a new branch for your contribution.
+2. Make your changes, ensuring that you follow our coding conventions.
+3. Write tests for your changes, if applicable.
+4. Submit a pull request with a clear and detailed description of your changes.
+
+## Code Style
+
+We use [The Black Code Style](https://black.readthedocs.io/en/stable/the_black_code_style/index.html).
+`black` and `isort` are used as formatters.
+
+## Code Linters
+
+We use almost default configuration of [flake8](https://flake8.pycqa.org/)
```

Comment:
@knopki why not exactly default? :thinking:

Reply:
We set line length to 88 and something more for compatibility with The Black Code Style. Also, relaxing some overly strict rules.